test/utils.h

```diff
@@ -137,28 +137,36 @@ inline TensorView* loweredTv(TensorView* tv, GpuLower& gpulw) {
 class PredicatedChecker : public kir::IrVisitor {
  public:
   // Checks if the provided tv is written to within a non-trivial conditional
-  static bool isPredicated(TensorView* tv, GpuLower& gpulw) {
-    PredicatedChecker checker(
-        loweredTv(tv, gpulw), gpulw.kernel()->topLevelExprs());
+  static bool isPredicated(StmtNameType tv_name, GpuLower& gpulw) {
+    PredicatedChecker checker(tv_name, gpulw.kernel()->topLevelExprs());
     return checker.is_predicated_;
   }
 
-  static bool isPredicated(TensorView* tv, kir::Kernel* kernel) {
-    PredicatedChecker checker(loweredTv(tv, kernel), kernel->topLevelExprs());
+  static bool isPredicated(StmtNameType tv_name, kir::Kernel* kernel) {
+    PredicatedChecker checker(tv_name, kernel->topLevelExprs());
     return checker.is_predicated_;
   }
 
+  static bool isPredicated(TensorView* tv, GpuLower& gpulw) {
+    return isPredicated(tv->name(), gpulw);
+  }
+
+  static bool isPredicated(TensorView* tv, kir::Kernel* kernel) {
+    return isPredicated(tv->name(), kernel);
+  }
+
  private:
   PredicatedChecker() = delete;
 
-  PredicatedChecker(TensorView* tv, std::vector<Expr*> exprs) : tv_(tv) {
+  PredicatedChecker(StmtNameType tv_name, std::vector<Expr*> exprs)
+      : tv_name_(tv_name) {
     kir::IrVisitor::handle(exprs);
   }
 
   using kir::IrVisitor::handle;
   bool is_predicated_ = false;
   bool predicated_ite_ = false;
-  TensorView* tv_ = nullptr;
+  StmtNameType tv_name_ = 0;
```

Comment:
Would fusion managed data make more sense?

Reply:
I don't see any particular reason as this is just a quick utility for test validations.

Reply:
When copying a fusion, do we copy the name of IR nodes, or do we reallocate names, and these names coincide?

Reply:
Oh, great. I thought they were just coincide. That's why I felt worried about this approach.